aws/policy/security-services.yaml

```diff
@@ -89,6 +88,7 @@ Statement:
       - logs:List*
       - secretsmanager:Describe*
       - secretsmanager:List*
+      - secretsmanager:GetRandomPassword
```

Comment:
Small nit, but could you make sure this is ordered alphabetically?

Reply:
sure.